docs/reference/modules/apostrophe-images/README.md

```diff
@@ -11,6 +11,23 @@ a website, and to manage a library of them for reuse.
 Each `apostrophe-image` doc has an `attachment` schema field, implemented
 by the [apostrophe-attachments](/reference/modules/apostrophe-attachments) module.
 
+::: warning NOTE
```

Comment:
All of this will be replaced the next time the documentation reference is generated. This should all go in the core code as comments. Search for one of the lines above (e.g., "Each `apostrophe-image` doc has an `attachment` schema field, implemented") in the core code to find where this is added. It will then be generated into the documentation.

Reply:
Ok, moved into core !